lug-helper.sh

```diff
@@ -2023,7 +2033,7 @@ version_menu(){
 # Save exported keybinds, wipe the USER directory, and restore keybinds
 rm_userdir() {
     # Prompt user to back up the current keybinds in the game
-    message info "Before proceeding, please be sure you have exported\nyour Star Citizen keybinds from within the game.\n\nTo do this, launch the game and go to:\nOptions->Keybindings->Control Profiles->Save Control Settings\n\nGo on; I'll wait."
+    message info "Before proceeding, please be sure you have exported\nyour Star Citizen keybinds and characters from within the game.\n\nTo do this, launch the game and go to:\nOptions->Keybindings->Control Profiles->Save Control Settings\n\nTo export your character, go to the character creator from the main menu and save it with a name.\n\nGo on; I'll wait."
```

Comment:
Note to self: I want to see how the word wrapping looks here before we merge and publish a new release